tutorials/Native_distributions_and_local_execution/jpackage-conveyor-comparison.md

```diff
@@ -0,0 +1,25 @@
+# Packaging tools comparison 
+
+Use [Conveyor](https://www.hydraulic.software) if you want any of the following features:
+
+* Online updates. jpackage creates packages that users must update manually by reinstalling.
+* Creation, signing and notarization of packages for all supported targets from any OS (e.g. your developer laptop, a CI machine). jpackage must be run on each platform you wish to support.
+* Self-signed packages with `curl | bash` style installs, which don't require you to purchase signing certificates.
+* Better support for JNI native libraries: 
+  * jpackage doesn't know how to extract or sign native libraries bundled into library JARs, which can cause macOS notarization failures. 
```

Comment:
Our plugin signs native libraries just fine. Again, I would prefer "Compose Multiplatform [Gradle] plugin" or something like that instead of `jpackage`.

Reply:
OK, thanks. I've removed this line from the PR already. I wonder what was going wrong for this other user (they posted about it in slack).